IdentityCore/src/util/NSString+MSIDTelemetryExtensions.m

```diff
@@ -73,4 +73,54 @@ - (NSDictionary *)msidParsedClientTelemetry
     return telemetryDict;
 }
 
++ (NSString *)msidAddPlatformSequenceParamWithName:(NSString *)name version:(NSString *)version toSequence:(NSString *)sequence
+{
+    NSMutableArray *sequenceItems;
+    static const int kSequenceItemsCount = 6;
+    static NSString *const kDelimeter = @",";
+    
+    if ([NSString msidIsStringNilOrBlank:sequence])
+    {
+        sequenceItems = [NSMutableArray new];
+        [sequenceItems addObject:@""]; //Src-SKU
+        [sequenceItems addObject:@""]; //Src-SKU-Ver
+        [sequenceItems addObject:@""]; //Msal-Runtime-Ver
+        [sequenceItems addObject:@""]; //Browser-Ext-Sku
+        [sequenceItems addObject:@""]; //Browser-Ext-Ver
+        [sequenceItems addObject:@""]; //BrowserCore-Ver
+    }
+    else
+    {
+        sequenceItems = [[sequence componentsSeparatedByString:kDelimeter] mutableCopy];
+    }
+    
+    // Validate count.
+    if (sequenceItems.count != kSequenceItemsCount)
+    {
+        MSID_LOG_WITH_CTX(MSIDLogLevelError,nil, @"Failed to add add platfrom sequence param: sequence count %d is invalid.", kSequenceItemsCount);
+        return nil;
+    }
+    
+    // Validate name.
+    if ([NSString msidIsStringNilOrBlank:name])
```

Comment:
Like 'Msal-Runtime-Ver' What if the sequenceItem is defined only by a version and has no name?

Reply:
We only support first 2 params in this method. Msal runtime will need to created their own logic.